src/Dropdown/Dropdown.js

```diff
@@ -19,23 +19,60 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   constructor() {
     super();
     this.state = {
+      menuDirection: 'down',
+      menuHeight: null,
       isOpen: false,
-      selectedID: null
+      selectedID: null,
+      knowMenuHeight: false
```

Comment:
Yeah, initially that's what I checked for. I added the separate `knowMenuHeight` variable because we may want to recalculate the height of the menu if the items inside of it are dynamic. But we're not doing that now — would you prefer to just remove the `knowMenuHeight` prop?